app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
@lowjoel sorry can you elaborate on how I can render the collection here? Was trying to figure out but I can't find a way to do it, because certain `course_users` might not have attempted the assessment (so no submission). 

The closest I got was to render an array as follows `[submission1, submission2, course_user1, submission3, course_user2]` and then rails can automatically infer the partial to render. But I don't like that approach because: (i) it still incurs the cost of building the array, and (ii) also result in us having to name the 'empty submission' partial as `_course_user.html.slim` which is potentially confusing. 

One other way is the following: 
```ruby
- submitted_users = submissions.map(&:course_users)
- non_attempted_users = course_students - submitted_users

- submissions.each do |submission|
  = render partial: 'submission', object: submission
- non_attempted_users.each do |user|
  = render partial: 'no_submission', locals: { course_user: course_user }
```

Is this what you were trying to get at?